tests/conformance/test_conformance.py

```diff
@@ -0,0 +1,699 @@
+    ],
+    "storage.objects.insert": [
+        blob_upload_from_string,
+        blob_upload_from_file,
+        blob_upload_from_filename,
+        blobwriter_write,
+        blob_create_resumable_upload_session,
+    ],
+    "storage.objects.patch": [blob_patch],
+    "storage.objects.rewrite": [blob_rewrite, blob_update_storage_class],
+    "storage.objects.update": [blob_update],  # S2 end
+}
+
+
+########################################################################################################################################
+### Pytest Fixtures for Populating Resources ############################################################################################
+########################################################################################################################################
+
+
+@pytest.fixture
+def client():
+    host = os.environ.get(STORAGE_EMULATOR_ENV_VAR)
+    client = storage.Client(client_options={"api_endpoint": host})
+    return client
+
+
+@pytest.fixture
+def bucket(client):
+    bucket = client.bucket(uuid.uuid4().hex)
+    client.create_bucket(bucket)
+    yield bucket
+    try:
+        bucket.delete(force=True)
+    except Exception:  # in cases where resources are deleted within the test
+        pass
+
+
+@pytest.fixture
+def object(client, bucket):
+    bucket = client.get_bucket(bucket.name)
+    blob = bucket.blob(uuid.uuid4().hex)
+    blob.upload_from_string("hello world", checksum="crc32c")
+    blob.reload()
+    yield blob
+    try:
+        blob.delete()
+    except Exception:  # in cases where resources are deleted within the test
+        pass
+
+
+@pytest.fixture
+def notification(client, bucket):
+    bucket = client.get_bucket(bucket.name)
+    notification = bucket.notification()
+    notification.create()
+    notification.reload()
+    yield notification
+    try:
+        notification.delete()
+    except Exception:  # in cases where resources are deleted within the test
+        pass
+
+
+@pytest.fixture
+def hmac_key(client):
+    hmac_key, secret = client.create_hmac_key(
+        service_account_email=_CONF_TEST_SERVICE_ACCOUNT_EMAIL,
+        project_id=_CONF_TEST_PROJECT_ID,
+    )
+    yield hmac_key
+    try:
+        hmac_key.state = "INACTIVE"
+        hmac_key.update()
+        hmac_key.delete()
+    except Exception:  # in cases where resources are deleted within the test
+        pass
+
+
+########################################################################################################################################
+### Helper Methods for Emulator Retry API ##############################################################################################
+########################################################################################################################################
+
+
+def _create_retry_test(host, method_name, instructions):
+    """
+    Initialize a Retry Test resource with a list of instructions and an API method.
```

Comment:
I've updated the docstrings. Hopefully they're now more informative and clear. 

As for the method name, I agree `_create_retry_test` isn't as self-explanatory as it could be. Though it's sort of symmetrical to the other helper method names😂  I could change it to `_send_retry_test_instructions` WDYT?